hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -179,17 +189,18 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
 
       setParameters(newMetadataLocation, tbl);
 
-      if (base != null) {
+      Table newTable = tbl;
```

Comment:
Why rename the variable here?

Reply:
Otherwise I can not use it to the lambda expressions. We need "final" variables in lambda calls.